sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DistinctQueryType.java

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.query;
+
+public enum DistinctQueryType {
+    // This means that the query does not have DISTINCT.
+    None,
```

Comment:
Java code style for enum. If the service sends the string value of these in the query plan you can utilize the pattern I added in the ConsistencyLevel

Reply:
Thankyou. Changed to follow java code style